tools/test_utils/src/MakeMeshes.cpp

```diff
@@ -251,8 +253,8 @@ MakeRectangularMeshForApiTesting(
     auto num_x = num_columns + static_cast<size_t>(1);
```

Comment:
hmmm :)

Reply:
No idea why this is like this.
I have changed it.